src/helpers/LandFunction.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+
+const today = () => {
+  const weekday = new Array(7);
+  weekday[0] = 'Sunday';
+  weekday[1] = 'Monday';
+  weekday[2] = 'Tuesday';
+  weekday[3] = 'Wednesday';
+  weekday[4] = 'Thursday';
+  weekday[5] = 'Friday';
+  weekday[6] = 'Saturday';
+  const date = weekday[new Date().getDay()];
+  return date;
+};
+const weather = (component) => {
+  let data;
+  const url = 'http://api.openweathermap.org/data/2.5/weather?q=kigali&APPID=baa9c286c4fa69c51bcf85107a7ac671';
+  axios.get(url)
+    .then((response) => {
```

Comment:
Could you use async/await here?

Reply:
why do you suggest async?

Reply:
It looks cleaner that way in my opinion.

Reply:
I think it will take me a lot of time but next time 
I will make sure to use it and will want you to help me with the issues I get sometimes with those async/await

Reply:
ohh I understand, great job @Ugizwenayo-Divine